src/GitTools.IssueTrackers/Context/IssueTrackerContext.cs

```diff
@@ -1,29 +0,0 @@
-﻿namespace GitTools.IssueTrackers
-{
-    public class IssueTrackerContext : Disposable, IIssueTrackerContext
-    {
-        public IssueTrackerContext()
```

Comment:
Temporary files such as locally checked out files / temp files we create during the run / etc

For example we have a workingdirectory that needs to be deleted.